tests/SprykerEcoTest/Zed/ComputopApi/Business/Mapper/ExpressCheckout/AbstractPayPalExpressMapperTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEcoTest\Zed\ComputopApi\Business\Mapper\ExpressCheckout;
+
+use Codeception\TestCase\Test;
+use SprykerEco\Shared\ComputopApi\Config\ComputopApiConfig;
+use SprykerEcoTest\Zed\ComputopApi\Business\Mapper\CreditCard\CreditCardMapperTestConstants;
+
+abstract class AbstractPayPalExpressMapperTest extends Test
+{
+    /**
+     * Return needed mapper
+     *
+     * @return \SprykerEco\Zed\ComputopApi\Business\Mapper\ExpressCheckout\PayPalExpressMapperInterface
+     */
+    abstract protected function createMapper();
+
+    /**
+     * @var \SprykerEcoTest\Zed\ComputopApi\Business\Mapper\ExpressCheckout\PayPalExpressMapperTestHelper
+     */
+    protected $helper;
+
+    /**
+     * @param \SprykerEcoTest\Zed\ComputopApi\Business\Mapper\ExpressCheckout\PayPalExpressMapperTestHelper $helper
+     *
+     * @return void
+     */
+    protected function _inject(PayPalExpressMapperTestHelper $helper): void
+    {
+        $this->helper = $helper;
+    }
+
+    /**
+     * @return void
+     */
+    public function testBuildRequest()
```

Comment:
Usually, the test name should consist of the called method name + [optional] input parameters description + expected outcome.